cyral/resource_cyral_repository_identity_map.go

```diff
@@ -312,7 +312,7 @@ func resourceRepositoryIdentityMap(deprecationMessage string) *schema.Resource {
 			},
 		},
 		Importer: &schema.ResourceImporter{
-			State: schema.ImportStatePassthrough,
+			StateContext: schema.ImportStatePassthroughContext,
```

Comment:
I believe we will need to create a custom import function for this resource, and then include a small section in the doc to explain how to import this resource (like we did for the other resources with complex IDs), as it depends on a lot of fields like repository_id, identity_type, identity_name, repository_local_account_id. So we could instruct users to import passing a combination of these fields and then we can split them.